.github/workflows/contracts.yml

```diff
@@ -203,7 +205,62 @@ jobs:
         run: |
           echo //registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }} > .npmrc
           npm publish --access=public
-      
+
+  contracts-build-and-publish-relay-maintainer-initcontainer:
```

Comment:
Would it be possible to move the relay maintainer job to a separate workflow file? Maybe it could be a separate step in the release process?

Reply:
According to our discussion off-GH, I moved the job to the `relay.yml` workflow.
Contrary to what we initially were thinking about, I decided not to use the `environment` input. I did that because we were using that input only to get the value of `GCR_REGISTRY_URL` and `GOOGLE_PROJECT_ID` variables (via `Load environment variables` step). Those variables have the same values for `ropsten` and `alfajores`, so it wouldn't really matter which environment is chosen in the input. So I removed the input and decided to use the `ropsten` config in the `Load environment variables` step.